cmake/modules/ifm3d-config.cmake.in

```diff
@@ -27,6 +27,13 @@ else()
           else()
             find_package(OpenCV REQUIRED COMPONENTS core)
           endif()
+		  find_package(OpenCV 3 REQUIRED COMPONENTS core)
```

Comment:
Are you sure that we should look for OpenCV 3 here? I would have expected that this is captured by the if above. BTW, do you know where OpenCV 2.4 is used? If this is Ubuntu 14.04 or 16.04 we probably can remove the whole check.

Reply:
Looks like Action responded already :-)

```
CMake Error at modules/image/src/libifm3d_image/CMakeLists.txt:9 (find_package):
  Could not find a configuration file for package "OpenCV" that is compatible
  with requested version "3.0".

  The following configuration files were considered but not accepted:

    /usr/share/OpenCV/OpenCVConfig.cmake, version: 2.4.9.1



-- Configuring incomplete, errors occurred!
See also "/home/runner/work/ifm3d/ifm3d/build/CMakeFiles/CMakeOutput.log".
See also "/home/runner/work/ifm3d/ifm3d/build/CMakeFiles/CMakeError.log".
Error: Process completed with exit code 1.
```

Reply:
For the backlog we should also add Ubuntu 20.04 to this list: https://github.com/ifm/ifm3d/blob/master/.github/workflows/ubuntu.yml#L15